fluentd/configs.d/openshift/output-applications.conf

```diff
@@ -0,0 +1,5 @@
+<match **>
+   @type copy
+   @include output-es-config.conf
+   @include ../dynamic/es-copy-config.conf
```

Comment:
@ewolinetz ok - then I think this commit should include the ability to provide `output-extra-*` and `output-extra-ops-*`, so that customers can use the plugins we already include, and we should implement support for out_secure_forward in a separate commit